sda/cmd/reencrypt/reencrypt.go

```diff
@@ -57,9 +53,9 @@ func (s *server) ReencryptHeader(_ context.Context, in *re.ReencryptRequest) (*r
 	newReaderPublicKeyList := [][chacha20poly1305.KeySize]byte{}
 	newReaderPublicKeyList = append(newReaderPublicKeyList, newReaderPublicKey)
 
-	log.Debugf("crypt4ghkey: %v", *Conf.ReEncrypt.Crypt4GHKey)
+	log.Debugf("crypt4ghkey: %v", *s.c4ghPrivateKey)
```

Comment:
maybe base64 encoded it, not sure how nice to show in logs as byte

Reply:
`%v` will format, but possibly `%x` gives a nicer output (`[104 101 106]` for `%v` vs `[68656a]` for `%x`).